src/main/webapp/app/exercises/programming/manage/build-plan-editor.component.ts

```diff
@@ -52,7 +54,7 @@ export class BuildPlanEditorComponent implements AfterViewInit, OnInit {
     ngAfterViewInit(): void {
         this.editor.getEditor().setOptions({
             animatedScroll: true,
-            maxLines: 80,
+            maxLines: 45,
```

Comment:
How is this change related to the PR? :)

Reply:
While working on this page I noticed that even in full-HD-fullscreen the box was extending too far. This was the easiest fix for that without having to dig into how the editor box can be dynamically resized (IIRC the online code editor also does not resize automatically, but in this case some manual dragging is possible to manually change the size, could be done as a separate PR in the future).